en/app.ftl

```diff
@@ -394,6 +398,22 @@ profile-filter-category-option-domain-based-aliases-v2 = Custom aliases
 # Filter on Relay aliases that only forward critical emails.
 profile-filter-category-option-critical-only-aliases = Critical-only aliases
 
+## Alias Promotional Email Blocking (displayed on the profile page)
+profile-promo-email-blocking-title = What emails do you want to block?
+# Block all emails sent to a speciic alias
+profile-promo-email-blocking-option-all = All
+# Block promotional emails sent to a speciic alias
+profile-promo-email-blocking-option-promotionals = Promotionals
+# Allow/forward all emails sent to a speciic alias
+profile-promo-email-blocking-option-none = None
+profile-promo-email-blocking-description-all = { -brand-name-relay } is blocking all emails sent to this alias.
+profile-promo-email-blocking-description-promotionals = { -brand-name-relay } will attempt to block promotional emails, while still forwarding emails like receipts and shipping information.
```

Comment:
Remove `,` before `while still forwarding emails... `

Reply:
Revised in 0c5106c652527ae5b78cf3ac22ed24b14977fc5e